docs/changelog.rst

```diff
@@ -1,5 +1,21 @@
 **Changelog**
 
+0.2.2
+^^^^^
+Monday Jul 12 2021
+
+* This version has been tested and required revisions, due to all actions used:
+
+  * ceph-osd: action `get-availability-zone`: **cs:~/rgildein/ceph-osd-0** (`LP#1911006`_)
+  * ceph-mon: action `get-health`: **cs:ceph-mon-48**
+  * ceph-mon: action `list-pools` with additional `format` argument: **cs:~/rgildein/ceph-mon-3** (`LP#1920135`_)
+  * ceph-mon: action `get-quorum-status` **not published** (`LP#1917690`_)
+  * nova-compute: action `instance-count`: **cs:~openstack-charmers-next/nova-compute-562** (`LP#1911011`_)
+  * nova-compute: action `node-name`: **cs:~openstack-charmers-next/nova-compute-562** (`LP#1911011`_)
+  * nova-compute: action `list-compute-nodes`: **cs:~openstack-charmers-next/nova-compute-562** (`LP#1911013`_)
+
+* juju version 2.8.10 is required to obtain the host name of the unit
```

Comment:
Do we need to mention this in the changelog? Shouldn't this be part of the "getting-started" or "requirements" sections?

Besides, I think this requirement has not changed between `0.2.1` and `0.2.2`, so I'm not sure it would fit in the changelog (or at least not in this section but in a previous one when the hostname attribute from `juju status` was used).

Reply:
Ok, I will add this into "getting-started", but also into README.md file.